src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -908,6 +917,19 @@ public function addQuantityInformation(
         }
 
         if ($show_availability) {
+            // If the product is discontinued and is displayed, we display proper message
```

Comment:
just a minor suggestion (only my opinion :man_shrugging: ). I prefer using same keywords in code (I think everyone in prestashop are familiar with the keywords "disabled|enabled or "active|inactive", but not yet so familiar with the new keyword you just introduced "discontinued". Furthermore, I think product can be inactive for multiple reasons, not only discontinued, lets say temporarily disabled or so...). So the following should be more appropriate and easier to understand to anyone.

```suggestion
            // If the product is disabled, but still displayed, we display a proper message
```